### ISSUE TITLE
Add links to files in README.md

### ISSUE THREAD

**Issue by SmartToolFactory** [SmartToolFactory/Jetpack-Compose-Tutorials#5]
It would be nice to add links on github/README.md to specific file that tutorial points to. If anyone would like to do this it's very appreciated.

**Comment by WuMingke** [comment]
> It would be nice to add links on github/README.md to specific file that tutorial points to. If anyone would like to do this it's very appreciated.

![image](https://github.com/SmartToolFactory/Jetpack-Compose-Tutorials/assets/22768714/1d8f6693-5817-4aa2-9a08-d3be9d92875e)

is this what you want?
（md grammar -> [Column](https://github.com/SmartToolFactory/Jetpack-Compose-Tutorials/blob/master/Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/chapter1_basics/Tutorial1_1ColumnRowBoxScreen.kt) ）

i am learning Compose from the project,
and thanks a lot .
I am glad you accepted my pr before.
i can add some links in my spare time.

**Comment by SmartToolFactory** [comment]
No. I mean when you click a title, this can be changed to a link, in README.md it would be nice if it opens corresponding kotlin file.

<img width="902" alt="Screenshot 2023-11-23 at 15 05 31" src="https://github.com/SmartToolFactory/Jetpack-Compose-Tutorials/assets/35650605/9dce7a7a-0148-4068-88d7-d522b5f5a66a">

https://github.com/SmartToolFactory/Jetpack-Compose-Tutorials/blob/master/Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/chapter1_basics/Tutorial1_2ClickSurfacesAndClicks.kt can direct to file itself. This could let anyone to access code snippet faster.

**Comment by SmartToolFactory** [comment]
https://github.com/SmartToolFactory/Jetpack-Compose-Tutorials/blob/master/Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/chapter1_basics/Tutorial1_2ClickSurfacesAndClicks.kt can direct to file itself. This could let anyone to access code snippet faster.

**Comment by SmartToolFactory** [comment]
I mean clicking 1-2 Surface, Shape, Cickable header should open kotlin file.

**Comment by WuMingke** [comment]
> I mean clicking 1-2 Surface, Shape, Cickable header should open kotlin file.

yes，we should be thinking the same thing.
you can click the 'Column' in my previous reply，it will open the 'Tutorial1_1ColumnRowBoxScreen.kt'.
or maybe you just want to add the link to the main title ？
